test/functional/apps/discover/_discover.ts

```diff
@@ -133,12 +135,10 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         const newDurationHours = await PageObjects.timePicker.getTimeDurationInHours();
         expect(Math.round(newDurationHours)).to.be(26);
 
-        await retry.waitFor('doc table to contain the right search result', async () => {
+        await retry.waitFor('doc table containing the documents of the brushed range', async () => {
           const rowData = await PageObjects.discover.getDocTableField(1);
-          log.debug(`The first timestamp value in doc table: ${rowData}`);
-          const dateParsed = Date.parse(rowData);
-          // compare against the parsed date of Sep 20, 2015 @ 17:30:00.000 and Sep 20, 2015 @ 23:30:00.000
-          return dateParsed >= 1442770200000 && dateParsed <= 1442791800000;
+          log.debug(`The first timestamp value in doc table after brushing: ${rowData}`);
+          return prevRowData !== rowData;
```

Comment:
Changing the test to be independed of timezone, checking if brushing the data changes the presented documents